js-plugins/postcss/src/index.ts

```diff
@@ -53,6 +68,49 @@ export default function farmPostcssPlugin(
       async executor(param, context) {
         try {
           const sourceMapEnabled = context.sourceMapEnabled(param.moduleId);
+          const enablePostcssImport =
+            options.internalPlugins?.postcssImport || true;
+
+          if (enablePostcssImport) {
+            postcssPlugins.unshift(
+              atImport({
+                resolve: async (id, basedir) => {
+                  const publicFile = await checkPublicFile(id, userConfig);
+                  if (publicFile) {
+                    return publicFile;
+                  }
+
+                  try {
+                    const resolvedInfo = await context.resolve(
+                      {
+                        kind: 'import',
+                        importer: param.moduleId,
```

Comment:
I think the importer should be `baseDir` or `baseDir/xxx` here. For import  hierarchy that more that 2, it may have issues. For example, `@import '../index.css'`, and in `../index.css` it has `@import './stylecss'`. Then `@import './stylecss'` should be resolved from `../index.css` instead of `param.moduleId`